src/Styleguide/Components/Footer.tsx

```diff
@@ -34,8 +56,30 @@ const FooterContainer: React.SFC<FooterContainerProps> = props => {
           Buy
         </Sans>
         <Serif size="2">
-          <Link href="https://www.artsy.net/#">Buying from Galleries FAQ</Link>
-          <Link href="https://www.artsy.net/#">Buying from Auctions FAQ</Link>
+          <Link
+            onClick={() => {
+              props.mediator &&
+                props.mediator.trigger("open:auth", {
```

Comment:
These will obviously be replaced, first need to add a corresponding `mediator.on` and a lil' refactor in Force to make these modals available.

Will attempt to do that and test via `yarn link`, etc but that can be a follow up PR